app/views/benefits/removeCompanyBenefitStopDate.scala.html

```diff
@@ -18,7 +18,7 @@
 @import uk.gov.hmrc.urls.Link
 @import includes.{ptaHeader, cancelLink}
 @import uk.gov.hmrc.tai.forms.benefits.RemoveCompanyBenefitStopDateForm
-@import includes.forms.{inputRadioGroupNewStyling, submit}
+@import includes.forms.{inputRadioGroupStyling, submit}
```

Comment:
I think we could drop the 'styling' suffix entirely. It was originally referring to the newer radio button styling, as per 
http://govuk-elements.herokuapp.com/form-elements/#form-radio-buttons
.. but since we've now removed the old stuff I don't think we need it.

Reply:
I will address it in my next request